### DISCUSSION TITLE
Unable to flash Sonoff S26 R2 v2.6 (zigbee)

### DISCUSSION THREAD

**Discussion by turkeryildirim** [arendst/Tasmota#19835] [Q&A] [answered]
Hi, 
I've purchased 2 pair of  Sonoff s26 r2 and tried to flash with tasmota but both devices rejected to enter flash mode. I tried couple different USB-TTL sticks, switched RX, TX pins, used DC power supply etc. and  no success. After struggling couple hours i checked the chip and it looks like it's changed ESP to CC2652.

So, is it still possible to flash tasmota to this device? If yes, how ? 

Here are some pictures of the device and my connections.
https://imgur.com/a/l5nqUtB

**Comment by sfromis** [comment]
Tasmota cannot be flashed on devices not using Espressif chips, which generally excludes all end devices using Zigbee. To use such a device, you need a coordinator device to manage the Zigbee network, and you can run Tasmota on some coordinator devices. This is very different from Tasmota on each end device, but can still be local-only IoT.

**Reply by turkeryildirim** [reply]
It looks like if i purchased Wifi version of this device, most probably i'd end up ESP chip one. Thanks for the info.

**Reply by sfromis** [reply, marked as answer]
Yes. https://templates.blakadder.com/sonoff_S26R2.html
